### DISCUSSION TITLE
Free quota doesn't upgrade when emails are deleted

### DISCUSSION THREAD

**Discussion by bpincaro** [modoboa/modoboa#2188] [Q&A] [answered]
# Impacted versions
1.17.0

* OS Type: Debian/Ubuntu
* OS Version: 20.04.2 LTS (GNU/Linux 5.4.0-65-generic x86_64)
* Database Type: MySQL
* Database version: mysql  V15.1 Distrib 10.3.25-MariaDB
* Modoboa: 1.17.0
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce

# Current behavior
Quota keeps inflating even when emails are deleted

Configured my mail client, everything almost works as expected (can send and receive mails) except for the fact that the quota keeps inflating. Deleting emails doesn't free some space as it was supposed to. Tried also on the webmail platform, deleted the mails in quarantine there, everything stayed the same... quota haven't changed!!!!

Checked the DB, table admin_quota, it doesn't update on emails delete only on new mails received. Is it normal? I'm guessing this is the problem.

# Expected behavior
Freeing quota on emails deletion

# Video/Screenshot link (optional)

**Comment by kryskool** [comment, marked as answer]
hi  @bpincaro 

Hev you try to recompute [quota](https://modoboa.readthedocs.io/en/latest/manual_installation/dovecot.html?highlight=quota#forcing-recalculation)  ?

regards,

**Comment by bpincaro** [comment]
Hi @kryskool ,
no, hadn't done it yet...
Guess it worked running the command!!! All seems to be working for now!

Thanks.
